docs/Tutorials/OpenAPI.md

```diff
@@ -318,6 +506,8 @@ There are 5 simple property types: Integers, Numbers, Strings, Booleans, and Sch
 * [`New-PodeOABoolProperty`](../../Functions/OpenApi/New-PodeOABoolProperty)
 * [`New-PodeOASchemaProperty`](../../Functions/OpenApi/New-PodeOASchemaProperty)
 
+* [`New-PodeOAMultiTypeProperty`](../../Functions/OpenApi/New-PodeOASchemaProperty) Note: OpenAPI 3.1 only
```

Comment:
Delete the erroneous blank new-line above the bullet point, and wrap the "Note:" part in `(..)` brackets

Reply:
The `Note: ...` part isn't in brackets; this is mostly just to appear nicer